src/main/fc/init.c

```diff
@@ -206,6 +208,53 @@ static IO_t busSwitchResetPin        = IO_NONE;
 }
 #endif
 
+static void configureSPIAndQuadSPI(void)
+{
+#ifdef USE_SPI
+    spiPinConfigure(spiPinConfig(0));
+#endif
+
+    sensorsPreInit();
+
+#ifdef USE_SPI
+    spiPreinit();
+
+#ifdef USE_SPI_DEVICE_1
```

Comment:
I hope you are aware that from 4.1 on all of these will be defined for all targets?

Reply:
Yes.  The list of defines hasn't changed the code was merely re-located.